src/serverUtils.js

```diff
@@ -0,0 +1,416 @@
+const fs = require('fs');
```

Comment:
Still a question of whether we want to duplicate and expose all of this GQL server/utils logic.

Reply:
Its ok for now, eventually we ll consolidate them.